src/main/java/com/google/devtools/build/lib/sandbox/SandboxOptions.java

```diff
@@ -359,15 +362,32 @@ public ImmutableSet<Path> getInaccessiblePaths(FileSystem fs) {
   public boolean sandboxHermeticTmp;
 
   @Option(
-      name = "experimental_sandbox_memory_limit_mb",
-      defaultValue = "0",
+      name = "experimental_sandbox_limits",
```

Comment:
Instead of introducing a separate mechanism for sandboxed spawns, I wonder if it's better to unify this with the specified `resource_set` of the sandboxed action. Then just use `--experimental_sandbox_enforce_resources_regexp` to control for which actions to enforce this on.

cc @wilwell.

Reply:
> You cannot override the resource set of arbitrary actions

By this I assume you mean actions outside of your control - e.g. in build rules not owned by your project / [default starlark resources](https://github.com/bazelbuild/bazel/blob/67c7e7284dd18ef0c55ca0994b30b142645e2677/src/main/java/com/google/devtools/build/lib/analysis/starlark/StarlarkActionFactory.java#L107).

>  for most actions the default resource set is underestimated

While I don't think fast testing / iteration is a good reason to introduce this (whatever value specified here should be relatively stable without users having to tinker frequently), I can see a legitimate case for this in the event that resources are underestimated. Since the `ResourceManager` makes reservations purely off of estimated values (and kind-of makes a claim that the action will have some minimum amount of resources<sup>1</sup>), this serves as a mechanism to upper bound resources.
___
<sup>1</sup> Not entirely true since Bazel assumes it can use all the resources available to the machine and doesn't actively monitor actually available resources.

Reply:
Done